android-components/components/feature/tabs/src/test/java/mozilla/components/feature/tabs/toolbar/TabCounterToolbarButtonTest.kt

```diff
@@ -54,7 +54,8 @@ class TabCounterToolbarButtonTest {
             currentState = initialState
         }
 
-        override fun getLifecycle(): Lifecycle = lifecycleRegistry
+        override val lifecycle: Lifecycle
+            get() = lifecycleRegistry
```

Comment:
We should be able to avoid the explicit `get() =` and do something like `override val lifecycle: Lifecycle = lifecycleRegistry` or even `override val lifecycle = lifecycleRegistry` if we want to save space. That would be my preference but I don't think either are required changes